app/Domain/Goalcanvas/Controllers/DelCanvasItem.php

```diff
@@ -12,6 +12,7 @@
     class DelCanvasItem extends \Leantime\Domain\Canvas\Controllers\DelCanvasItem
     {
         protected const CANVAS_NAME = 'goal';
+
```

Comment:
Can you replace the controller code with the one from \Leantime\Domain\Canvas\Controllers\DelCanvasItem ? as well.
I'd like to de-couple goals entirely from the canvas module